README.md

```diff
@@ -296,9 +296,16 @@ tcpdump
 pip3 install --user -r requirements.txt
 ```
 
-For documentation building:
+**For documentation building:**
+
+**Tools**
 ```bash
 sudo apt-get install -y doxygen graphviz texlive-full
+```
+**Theme** 
+```bash
+git clone --depth 1 -b v2.3.3 https://github.com/jothepro/doxygen-awesome-css ./docs/doxygen/awesome_css
```

Comment:
I see. It might actually make sense to have this as a submodule then. I am torn between a cmake approach and just using regular submodules. We use CMake for a lot of things but it could be overkill for the documentation.

Reply:
I don't think many users will be building documentation locally, considering that we have hosted it now.